src/Bundle/BehatDoctrineDataFixturesExtensionBundle.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2018 Spomky-Labs
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+namespace BehatExtension\DoctrineDataFixturesExtension\Bundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class BehatDoctrineDataFixturesExtensionBundle extends Bundle
```

Comment:
I hope, most infrastructure classes could be finalized by default since there is no point to extend them

Reply:
Done for some classes.